src/components/dar_table/DarTableActions.js

```diff
@@ -111,7 +117,7 @@ export default function DarTableActions(props) {
           index,
           updateLists,
           isIcon: visibilityOptions.showCancelIcon,
-          isRendered: isElectionOpen(election),
+          isRendered: isElectionOpen(election) && consoleType !== 'member',
           disabled: !isChair && consoleType !== 'manageAccess'
```

Comment:
To prevent future bugs, we should treat these kinds of static string values as enums. Something like:
```
const consoleTypes = {
    MEMBER: 'member',
    MANAGE_ACCESS: 'manageAccess'
}
```
... etc. and then you can compare to those values, i.e.
```
consoleType !== consoleTypes.MEMBER;
```
The trick is to place `consoleTypes` in the right place. Not sure where that should live at the moment, but probably in this component.